solutions/usaco-715.mdx

```diff
@@ -98,4 +98,30 @@ public class MaxCross {
 ```
 
 </JavaSection>
+
+<PySection>
+
+```python
+with open('maxcross.in') as read:
+	n, k, b = map(int, read.readline().split())
+	id, left, right, tmp = [0] * (n + 1), 1, k, 0
+	
+	for _ in range(b):
+		id[int(read.readline())] = 1
+
+for i in range(left, right + 1):
+	tmp += id[i]
```

Comment:
tmp is a bad name, you literally use it throughout the entire program

Reply:
okay